ui/helpers/url.js

```diff
@@ -144,6 +144,12 @@ export const getPerfAnalysisUrl = function getPerfAnalysisUrl(url, job) {
   return profilerUrl;
 };
 
+export const getCrashViewerUrl = function getCrashViewerUrl(crashJsonUrl) {
+  return `https://fqueze.github.io/aretestsfastyet/crash-viewer.html?url=${encodeURIComponent(
```

Comment:
Are there plans to host this with a Mozilla account?

Reply:
There are plans, yes. The crash-viewer is a standalone HTML file so it could go anywhere. The dashboards from the same repository that need JSON artifacts to be produced daily need a bit more thinking (I'll try to have taskcluster produce them).